deployments/data101/image/environment.yml

```diff
@@ -77,6 +77,7 @@ dependencies:
 - statsmodels==0.14.0
 - sympy==1.10.1
 - tornado==6.2.0
+- trifacta==8.5.1
```

Comment:
hey @franklsf95 !  there's no conda package for this, only pypi.  move the package req from the `conda` block down to the `pip` block at the end of the file.

Reply:
Oops, fixed.